compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/SkiaParagraph.skiko.kt

```diff
@@ -209,8 +211,7 @@ internal class SkiaParagraph(
     override fun isLineEllipsized(lineIndex: Int) = false
 
     override fun getLineForOffset(offset: Int) =
-        lineMetricsForOffset(offset)?.run { lineNumber.toInt() }
-            ?: 0
+        lineMetricsForOffset(offset)?.lineNumber ?: 0
```

Comment:
What result type ? 
Int or Long ?

Reply:
since `.toInt()` was "gray" in IDE and `override` still works - `Int`